cmd/kube-controller-manager/app/options/jobcontroller.go

```diff
@@ -32,6 +32,7 @@ func (o *JobControllerOptions) AddFlags(fs *pflag.FlagSet) {
 	if o == nil {
 		return
 	}
+        fs.Int32Var(&o.ConcurrentJobSyncs, "concurrent-job-syncs", o.ConcurrentJobSyncs,"The number of job objects that are allowed to sync concurrently. Larger number = more responsive jobs, but more CPU (and network) load")
```

Comment:
minor change:
`A larger number means more responsive jobs, but more CPU and network load`

Reply:
@neolit123  done

Reply:
i don't see your update.
did you forget to push?